bin/handler-sendgrid.rb

```diff
@@ -18,11 +18,11 @@ def short_name
   end
 
   def action_to_string
-   @event['action'].eql?('resolve') ? "RESOLVED" : "ALERT"
+    @event['action'].eql?('resolve') ? 'RESOLVED' : 'ALERT'
   end
 
   def default_config
-    #_ Global Settings
+    # _ Global Settings
```

Comment:
When I pull this in I will see if I can understand whatever rule this might have been there might be something fairly low level I am unfamiliar with. Looking forward to learning something new.